test/test.js

```diff
@@ -0,0 +1,25 @@
+const test = require('ava').test
+const fn = require('../covenant.js')
+const fs = require('mz/fs')
+const path = require('path')
+
+test.after('cleanup', t => {
+  fs.unlink(path.join(__dirname, 'fixtures/RES.md'))
+})
+
+test('throws if no email provided', async t => {
+  const error = await t.throws(fn())
+  t.is(error.message, 'An email address must be provided!')
+})
+
+test('will create file', async t => {
+  const dest = path.join(__dirname, 'fixtures/RES.md')
```

Comment:
If this fails to download a file, and the test has been run before, would this still pass because the file exists?

Reply:
Nope. `test.after('cleanup', cb)` removes it after the test runs.

Reply:
cool!